src/pages/EquipmentRental/index.tsx

```diff
@@ -1,39 +1,109 @@
 import React, { useEffect, useState } from "react";
+import axios from "axios";
 import { Card } from "../../components/Card";
-import { CardProps } from "../../utils/enum";
-import { fakeRentalData } from './fakeData';
+import { API_BASE_URL, CardProps, SelectOption } from "../../utils/enum";
+// import { fakeRentalData } from './fakeData';
 import { BarLoader } from "react-spinners";
 import { Select } from "../../components/Select";
 import { Button } from "../../components/Button";
 import { InputText } from "../../components/InputText";
+import { SelectMulti } from "../../components/SelectMulti";
 
 export const EquipmentRental = () => {
   const [loading, setLoading] = useState(true);
-  const [rentalData, setRentaldata] = useState<CardProps[]>(fakeRentalData);
+  // const [rentalData, setRentalData] = useState<CardProps[]>(fakeRentalData);
+  const [rentalData, setRentalData] = useState<CardProps[]>();
+
+  const [facilityName, setFacilityName] = useState('');
+  const [equipmentName, setEquipmentName] = useState('');
+  const [facilityOptions, setFacilityOptions] = useState<SelectOption[]>([{label: 'All', value: 'All'}]);
+
+
+  const handleFetch = async () => {
+
+    console.log(facilityName);
+
+    try {
+      const res = await axios({
+        baseURL: API_BASE_URL,
+        method: 'get',
+        url: '/rental',
+        params: {
+          facility: facilityName,
+          equipment: equipmentName
+        },
+        headers: { 'Content-Type': null, cache: false }
+      });
+      if (res.status === 200) {
+        if (res.data.equipment) setRentalData(res.data.equipment);
+        setLoading(false);
+      }
+    } catch (err) {
+      console.log(err);
+    }
+  };
+
+  const handleFetchOptions = async () => {
+    try {
+      const res = await axios({
+        baseURL: API_BASE_URL,
+        method: 'get',
+        url: '/rental/getOptions',
+        headers: { 'Content-Type': null }
+      });
+      if (res.status === 200) {
+        console.log('getOptions called');
+        if (res.data.facilityOptions) {
+          const fo: SelectOption[] = [];
+          res.data.facilityOptions.forEach((item: any) => {
```

Comment:
Since the actual select option component you're using on line 88 only takes an array of strings you can just literally set the facility options with the res.data.facilityOptions. Right now you're putting it into an array of objects, then converting that array of objects back into an array of strings using toStringArr